pages/general-information-and-resources/san-francisco.md

```diff
@@ -405,6 +392,18 @@ the health unit is 415-556-2766.
 
 The Health Unit also includes an all-gender bathroom.
 
+### Bike parking
+
+Bike parking is next to the loading dock, entered from the far corner at
+McAllister and Leavenworth.
+
+To request bike parking access: Fill out
+[this form](https://docs.google.com/document/d/18N-RSYzyQ2s9orhgsmIce2aOSKvayI8f433oMmI1Dw4/)
+and return to [James E.](mailto:james.eneliko@gsa.gov), either by email or by printing and returning the form to the building office in the basement level.
```

Comment:
Thanks for doing this!

As a side note, when I try to print the form to pdf so I can email it to my home email to print it out, the font size is so enormous it prints across 4+ pages. I didn't succeed in smallifying it, nor making it a fillable form. Seems like that should be straightforward. How do I do that?

Reply:
Hmmmmmm ... looks like the font size is 14pt in Google Docs, so I'm not sure why it's printing out huge. 🤔